test/java/seedu/addressbook/util/TypicalPersons.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.util;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList;
+
+/**
+ * Class to generate typical test persons
+ */
+public class TypicalPersons {
+
+    public static Person amy, bill, candy, donald, grace;
```

Comment:
@damithc says:
> The extra work caused by duplication of code is a good source of 'low hanging fruit' issues for new contributors to do.

It's more work for reviewers through, and will slow down the project. Reviewers may not be happy that they have to review the same things over and over again, but subtly different each time due to differences in code bases.

> at the moment I'm not too eager to unify similarities between the levels.

I understand. However I hope that until there is a clear direction on how the different levels are going to differ from each other, we try to prevent them from diverging unnecessarily, simply because it is *much* easier to make code bases diverge rather than converge.

For an example of "diverging unnecessarily", for this PR I hope that the test data is kept exactly the same as the other levels, because future tests will be written *for* the test data. Having test data that is different will just cause unnecessary friction in the future should we ever try to bring tests from one level to another. Just try to cover our bases here :-)